core/src/test/resources/applicationContext-dao.xml

```diff
@@ -17,6 +17,8 @@
 		     <list>
 			     <value>file:///#{systemEnvironment['PORTAL_HOME']}/src/main/resources/portal.properties</value>
                  <value>file:///#{systemEnvironment['PORTAL_HOME']}/src/main/resources/maven.properties</value>
+                 <value>file:///#{systemEnvironment['PORTAL_HOME']}/portal.properties</value>
```

Comment:
Why are these modifications necessary?

Reply:
These were added to resolve CI build failure - Could not resolve placeholder 'spark.app.name' in value "${spark.app.name}". An example of a failed build: https://api.travis-ci.org/v3/job/552881291/log.txt